include/glow/IR/IR.h

```diff
@@ -140,12 +149,12 @@ class Instruction : public Value {
   /// Adds a new operand \p op at the end of the operand list.
   void pushOperand(Operand op);
 
-  Instruction(IRFunction *M, llvm::StringRef name, Kinded::Kind k, TypeRef Ty)
-      : Value(name, Ty, k), F_(M) {}
+  Instruction(llvm::StringRef name, Kinded::Kind k, TypeRef Ty)
```

Comment:
Why did you remove the parent method and put nullptr instead?

Reply:
We were setting the parent in the constructor, before we even added the instruction to an IRFunction. Now the parent link is set when the instruction is added to the instruction list of a function for real.